src/models/api/FlowRunResponse.ts

```diff
@@ -38,4 +38,5 @@ export type FlowRunResponse = {
   work_queue_name: string | null,
   work_pool_name: string | null,
   work_pool_queue_name: string | null,
+  job_variables?: Record<string, unknown> | null,
```

Comment:
`?` for compatibility w/ OSS - open PR over there to add this field so atm this may or may not be there, but coalescing to `{}` in the mapper either way

Reply:
think we should be able to remove this `| null` also but gonna wait for the apis to stabilize

Reply:
Make sense, thanks!